assets/dev/scss/frontend/_container.scss

```diff
@@ -41,10 +41,6 @@
 	// Set `min-width` to fix nested Containers shrink bug (ED-4964).
 	// Ref: https://stackoverflow.com/questions/43934648/how-to-make-flexbox-items-shrink-correctly-when-in-a-nested-container
 	min-width: 0;
-	// Use the `min()` function to fix cases when a top-level Container is 100% width due to `max-width: 100%;`,
-	// or cases when a nested Container overflows the parent Container due to `max-width: 1140px;`.
-	// Capital `M` in order to use the CSS function and not the SCSS one.
-	max-width: Min( 100%, var( --width ) );
 	min-height: var( --min-height );
```

Comment:
I have removed the max-width from the 'general container settings' to the 'parent container styling'